content/ja/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -0,0 +1,443 @@
+---
+title: Podとコンテナにセキュリティコンテキストを設定する
+content_type: task
+weight: 80
+---
+
+<!-- overview -->
+
+セキュリティコンテキストはPod・コンテナの特権やアクセスコントロールの設定を定義します。
+セキュリティコンテキストの設定には以下のものが含まれますが、以下のものには限定されません。
+
+* 任意アクセス制御: [user ID (UID) と group ID (GID)](https://wiki.archlinux.org/index.php/users_and_groups)に基づいて、ファイルなどのオブジェクトに対する許可を行います。
+
+* [Security Enhanced Linux (SELinux)](https://ja.wikipedia.org/wiki/Security-Enhanced_Linux):
+  オブジェクトにセキュリティラベルを付加します。
+
+* 特権または非特権として実行します。
+
+* [Linux Capabilities](https://linux-audit.com/linux-capabilities-hardening-linux-binaries-by-removing-setuid/):
+  rootユーザーのすべての特権ではなく、一部の特権をプロセスに与えます。
+
+* [AppArmor](/docs/tutorials/security/apparmor/):
+  プロファイルを用いて、個々のプログラムのcapabilityを制限します。
+
+* [Seccomp](/docs/tutorials/security/seccomp/): プロセスのシステムコールを限定します。
+
+* `allowPrivilegeEscalation`: あるプロセスが親プロセスよりも多くの特権を得ることができるかを制御します。 この真偽値は、コンテナプロセスに
+  [`no_new_privs`](https://www.kernel.org/doc/Documentation/prctl/no_new_privs.txt)
+  フラグが設定されるかどうかを直接制御します。
+  コンテナが特権で動いているか、`CAP_SYS_ADMIN`を持っている場合、`allowPrivilegeEscalation`は常にtrueになります。
```

Comment:
```suggestion
  コンテナが以下の場合、`allowPrivilegeEscalation`は常にtrueになります。
  - コンテナが特権で動いている、または
  - `CAP_SYS_ADMIN`を持っている
```
I think it would be better to adapt the format to en-site.